sympy/sets/sets.py

```diff
@@ -218,12 +218,8 @@ def _complement(self, other):
 
         elif isinstance(other, FiniteSet):
             from sympy.utilities.iterables import sift
-
-            def ternary_sift(el):
-                contains = self.contains(el)
```

Comment:
leave ternary_sift in place but make it's 1 line contents be `return fuzzy_bool(self.contains(e1))`